### DISCUSSION TITLE
一分钟读论文：《StackOverflow 上 JS 代码片段规则违规的挖掘》

### DISCUSSION THREAD

**Discussion by giscus** [unbug/unbug.github.io#14] [Announcements]
# 一分钟读论文：《StackOverflow 上 JS 代码片段规则违规的挖掘》

还在复制粘贴 StackOverflow 的代码吗？巴西的巴西利亚大学和巴西贝伦联邦大学合著的论文《Mining Rule Violations in JavaScript Code Snippets》对 StackOverflow 的 336k 个代码片段进行分析，发现答案里 100% 的 JavaScript 代码都存在 ESLinter 不通过问题。

https://unbug.github.io/Mining-Rule-Violations-in-JavaScript-Code-Snippets/

**Comment by unbug** [comment]
StackOverflow 的 CTR+V的代码 100% 的 ESLinter 不通过
